modules/ting_fulltext/ting_fulltext.module

```diff
@@ -92,3 +102,32 @@ function ting_fulltext_ctools_plugin_directory($module, $plugin) {
     return "plugins/$plugin";
   }
 }
+
+/**
+ * Ajax call function wich returns the review in full text.
+ */
+function ting_fulltext_get_review($object_id) {
+  try {
+    $path = drupal_get_path('module', 'ting_fulltext');
+    // Include functions to parse the xml.
+    include_once $path . '/includes/ting_fulltext.pages.inc';
```

Comment:
Please use [`module_load_include()`](https://api.drupal.org/api/drupal/includes%21module.inc/function/module_load_include/7.x) for this.